tests/ha/ha_cluster_init.pm

```diff
@@ -56,7 +56,8 @@ sub run {
     my $corosync_conf = '/etc/corosync/corosync.conf';
     my $sbd_device = get_lun;
     my $sbd_cfg = '/etc/sysconfig/sbd';
-    my $unicast_opt = get_var("HA_UNICAST") ? '-u' : '';
+    my $unicast_arg = is_sle('>=15-SP6') ? '--transport udpu' : '-u';
```

Comment:
Please do a VR of 15-SP6,7 or change this to 16.

Reply:
Ticket: sle 16
My brain: sle15sp6 it is